src/rpcclient/tests/test_capture_fd.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import pytest
+
+from rpcclient.capture_fd import CaptureFD
+
+TEST_DATA: bytes = b"testtesttest"
+
+
+@pytest.mark.parametrize('fd', [
+    sys.stdout.fileno(),
+    sys.stderr.fileno()
+])
+def test_capture_fd(client, fd: int) -> None:
+    """
+    :param rpcclient.client.Client client:
+    :param int fd:
+    """
+    with CaptureFD(client, fd) as cap:
+        if -1 == client.symbols.write(fd, TEST_DATA, len(TEST_DATA)):
+            client.raise_errno_exception("write to fd failed")
+        data: bytes = cap.read()
```

Comment:
```suggestion
        data = cap.read()
```

Let implicit typing define it

Reply:
fixed

Reply:
sorry, fixed now